sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -3370,54 +3435,6 @@ class Analyzer(override val catalogManager: CatalogManager)
     }
   }
 
-  private def commonNaturalJoinProcessing(
```

Comment:
why do we move this method? It creates a lot of code diff and makes it harder to review.

Reply:
I can move it back - I just wasn't sure why it lived outside of this class, given that it's not shared.